.github/workflows/ci-pyright-fails.yml

```diff
@@ -34,4 +34,4 @@ jobs:
       - name: Pyright
         uses: jakebailey/pyright-action@v1
         with:
-          version: 1.1.300
+          version: 1.1.301
```

Comment:
Is this any helpful?
```
inputs:
  version:
    description: 'Version of pyright to run. If not specified, the latest version will be used.'
    required: false
```

Reply:
Yep, because the 1.1.299 -> 1.1.300 broke a lot of things